sdk/tests/conformance/textures/misc/tex-sub-image-2d-bad-args.html

```diff
@@ -107,6 +107,15 @@
     wtu.glErrorShouldBe(gl, gl.INVALID_OPERATION, "type not same as original RGBA 4_4_4_4");
 }
 
+var largeCanvas = document.createElement("canvas");
```

Comment:
I updated this pull request. Now it does the same checks as above code for small canvas.